roles/confluent.test/molecule/custom-user-plaintext-rhel/verify.yml

```
@@ -0,0 +1,68 @@
+---
+- name: Verify - Zookeeper User Change
+  hosts: zookeeper1-kafka-broker1
+  gather_facts: false
+  tasks:
+    - name: Set Zookeeper user to new user
+      set_fact:
+        zookeeper_user: zk-test
+
+    - name: Set Zookeeper Group
+      set_fact:
+        zookeeper_group: confluent
```

```suggestion
  vars:
    zookeeper_user: zk-test
    zookeeper_group: confluent
  tasks:
```

can apply vars to a playbook, and shorten the playbook by two tasks